index.mako

```diff
@@ -180,9 +180,9 @@
 </pre>
             <h2>Supported languages</h2>
             <ul>
-                <li><code>node</code></li>
-                <li><code>python</code></li>
-                <li><code>ruby</code></li>
+                <li><code>node</code> - must have a package.json (Installed via <code>npm install .</code>)</li>
+                <li><code>python</code> - must have a setup.py (Installed via <code>pip install .</code>)</li>
```

Comment:
I think it'd be good to at least mention the necessity for entry points to exist in `setup.py`, matching the ones in `hooks.yaml`.